packages/protocol/contracts/L1/TaikoL1.sol

```diff
@@ -80,7 +80,9 @@ contract TaikoL1 is
         (meta, depositsProcessed) =
             LibProposing.proposeBlock(state, config, AddressResolver(this), params, txList);
 
-        _verifyBlocks(config, config.maxBlocksToVerifyPerProposal);
+        if (!state.slotB.provingPaused) {
+            _verifyBlocks(config, config.maxBlocksToVerifyPerProposal);
+        }
```

Comment:
I dont think it is necessary. 🤔 
`_verifyBlocks()` has the modifier `whenProvingNotPaused`